spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'POSTS', type: :request do
+  describe 'GET POSTS #index' do
+    before(:example) { get('/users/15/posts#index') }
+
+    it 'server return 200 server ok' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'template rendering correctly' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'shows body content of index' do
+      expect(response.body).to include('<h1>show the posts of a user</h1>')
+    end
+  end
+
+  describe 'GET POSTS #index' do
+    before(:example) { get('/users/15/posts/20') }
```

Comment:
- [ ] There is one typo here, it should be `#show` instead of `#index` on line 20.

Reply:
Yes I copy pasted from "users" since it was almost the same and I've forgot that part. Thanks for mentioning it.
As for the readme. Since we're at the very beginning I thought I'd leave that for when it'll have more relevance but I'll add it now.
Thank you very much @Dipeshtwis. The next commits will have all the changes required